src/legacy/core_plugins/kibana/public/dashboard/panel/panel_utils.ts

```diff
@@ -44,10 +57,10 @@ export class PanelUtils {
       y: panel.row - 1,
       w: panel.size_x || DEFAULT_PANEL_WIDTH,
       h: panel.size_y || DEFAULT_PANEL_HEIGHT,
-      i: panel.panelIndex.toString()
+      i: panel.panelIndex.toString(),
     };
+
     panel.version = chrome.getKibanaVersion();
-    panel.panelIndex = panel.panelIndex.toString();
```

Comment:
Given what we've learned about `panelIndex` potentially being a `number` in older versions, think we should keep this line (sorry for the confusion I introduced by suggesting we remove it originally!)

Reply:
Added this line back in d3ad4ef